pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -323,6 +323,7 @@ func (r *ReconcileToolchainStatus) membersHandleStatus(logger logr.Logger, toolc
 			logger.Error(err, fmt.Sprintf("cannot find memberstatus resource in namespace %s in cluster %s", memberCluster.OperatorNamespace, memberCluster.Name))
 			memberStatusNotFoundCondition := status.NewComponentErrorCondition(toolchainv1alpha1.ToolchainStatusMemberStatusNotFoundReason, err.Error())
 			memberStatus := customMemberStatus(*memberStatusNotFoundCondition)
+			memberStatus.ApiEndpoint = memberCluster.APIEndpoint
```

Comment:
Since the APIEndpoint is taken from  `ToolchainCluster` CR here in host-operator, then I guess we should move the parameter from `MemberStatus` CRD into `Member` type in `ToolchainCluster` CRD: https://github.com/codeready-toolchain/api/blob/master/pkg/apis/toolchain/v1alpha1/toolchainstatus_types.go#L180-L190
The reason is that `MemberStatus` CRD is used in member operator to give information about the state of the member cluster to the host operator - that information is then copied into `ToolchainStatus` CR. But because we don't need that parameter in member operator at all, we should remove it from the `MemberStatus` CRD

Reply:
It's not like we don't need it in member (it could be a good place to store it there but we don't have to) we rather don't have it in member. I didn't realize we do a ToolchainStatus.Status.Member[] which wraps individual MemberStatuses though.. So, yeah, +1 to moving it to https://github.com/codeready-toolchain/api/blob/master/pkg/apis/toolchain/v1alpha1/toolchainstatus_types.go#L180-L190

Reply:
moved apiEndpoint to Member in https://github.com/codeready-toolchain/api/pull/221

Reply:
added the change now, @alexeykazakov and @MatousJobanek